platform/pulpcore/app/status.py

```diff
@@ -0,0 +1,65 @@
+"""
+Module to return status information about a running Pulp instance
```

Comment:
This module was copied over from server/pulp/server/managers/status.py, which separates the helper functions from the view (server/pulp/server/webservices/views/status.py). I have no problem with moving these functions into the view though.